custom/intrahealth/sqldata.py

```diff
@@ -6118,6 +6138,14 @@ def _get_amount_owed(ppses_data):
 
         return {'html': to_return}
 
+    @staticmethod
+    def _collect_value(values_dict, key, product_value):
+        value = product_value['html']
+        if values_dict.get(key, None):
+            values_dict[key] += value
+        else:
+            values_dict[key] = value
```

Comment:
You seem to use this pattern a lot.  You should read up on python's `defaultdict`, which will allow you to reduce this type of boilerplate.

```python
In [1]: from collections import defaultdict

In [2]: values = defaultdict(int)

In [3]: values['a'] += 1

In [4]: values['b'] += 1

In [5]: values['a'] += 3

In [6]: values
Out[6]: defaultdict(int, {'a': 4, 'b': 1})
```